### ISSUE TITLE
Support patches with multiple versions

### ISSUE THREAD

**Issue by mpchadwick** [philwinkle/Philwinkle_AppliedPatches#17]
SUPEE-7405 has both a v1 and a v1.1

SUPEE-7405 v1

```
2016-01-16 16:39:38 UTC | SUPEE-7405-EE-1-13-1-0 | EE_1.13.1.0 | v1 | 6473815ee7783aa55921e6c27e0eb8c2ce82cfc4 | Tue Jan 19 14:35:31 2016 +0200 | 6da102a0a8..6473815ee7
patching file app/code/core/Enterprise/Checkout/Block/Adminhtml/Manage/Form/Coupon.php
patching file app/code/core/Enterprise/PageCache/etc/config.xml
// Many more files follow
```

SUPEE-7405 v1.1

```
2016-02-25 18:23:26 UTC | SUPEE-7405 | EE_1.13.1.0 | v1.1 | 97d160644714ce235a453ccf8ebdaae6033ef65d | Fri Feb 5 12:26:33 2016 +0200 | 6473815ee7783aa55921e6c27e0eb8c2ce82cfc4..97d160644714ce235a453ccf8ebdaae6033ef65d
patching file app/code/core/Mage/Adminhtml/Helper/Sales.php
patching file app/code/core/Mage/Core/Model/Config.php
patching file app/code/core/Mage/Sales/Model/Quote/Item.php
patching file lib/Varien/File/Uploader.php
```

Should this module have a method for showing that multiple versions of a single patch have been applied?

**Comment by philwinkle** [comment]
Absolutely. I see your PR against this so I'll review over there.

**Comment by philwinkle** [comment]
Fixed in #18
